packages/blockchain-wallet-v4-frontend/src/modals/Wallet/InterestPromo/template.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 import { FormattedMessage } from 'react-intl'
-import { calcBasicInterest } from 'blockchain-wallet-v4-frontend/src/modals/Earn/Interest/conversions'
```

Comment:
It would be great to clean up `conversions.ts` since some of the functions from it do not seems to be used anywhere.